api/db/repositories/customerRepository.js

```diff
@@ -47,9 +43,9 @@ exports.getAll = async () => {
 
     if (allCustomers === null || allCustomers === undefined) { 
         throw new Error('Customers table is empty. REPO');
+    } else {
+        return allCustomers;
```

Comment:
there is no need for `else` block, because if that `if (allCustomers === null || allCustomers === undefined)` is gonna be true it's gonna throw new error and it's gonna get out of the function. So `return allCustomers;` can stay out of else block

Reply:
Affirmative